src/PENIntegracao.php

```diff
@@ -5,146 +5,153 @@
         $strAcoesProcedimento .= '<a id="validar_expedir_processo" href="' . $objPaginaSEI->formatarXHTML($objSessaoSEI->assinarLink('controlador.php?acao=pen_procedimento_expedir&acao_origem=procedimento_visualizar&acao_retorno=arvore_visualizar&id_procedimento=' . $dblIdProcedimento . '&arvore=1')) . '" tabindex="' . $numTabBotao . '" class="botaoSEI"><img class="infraCorBarraSistema" src=' . ProcessoEletronicoINT::getCaminhoIcone("/pen_expedir_procedimento.gif", $this->getDiretorioImagens()) . ' alt="Envio Externo de Processo" title="Envio Externo de Processo" /></a>';
       }
     }
 
-      //Apresenta o bot�o da p�gina de recibos
+    //Apresenta o bot�o da p�gina de recibos
     if($bolAcaoExpedirProcesso){
-        $objProcessoEletronicoDTO = new ProcessoEletronicoDTO();
-        $objProcessoEletronicoDTO->retDblIdProcedimento();
-        $objProcessoEletronicoDTO->setDblIdProcedimento($dblIdProcedimento);
-        $objProcessoEletronicoRN = new ProcessoEletronicoRN();
+      $objProcessoEletronicoDTO = new ProcessoEletronicoDTO();
+      $objProcessoEletronicoDTO->retDblIdProcedimento();
+      $objProcessoEletronicoDTO->setDblIdProcedimento($dblIdProcedimento);
+      $objProcessoEletronicoRN = new ProcessoEletronicoRN();
       if($objProcessoEletronicoRN->contar($objProcessoEletronicoDTO) != 0){
-          $numTabBotao = $objPaginaSEI->getProxTabBarraComandosSuperior();
-          $strAcoesProcedimento .= '<a href="' . $objSessaoSEI->assinarLink('controlador.php?acao=pen_procedimento_estado&acao_origem=procedimento_visualizar&acao_retorno=arvore_visualizar&id_procedimento=' . $dblIdProcedimento . '&arvore=1') . '" tabindex="' . $numTabBotao . '" class="botaoSEI">';
-          $strAcoesProcedimento .= '<img class="infraCorBarraSistema" src=' . ProcessoEletronicoINT::getCaminhoIcone("/pen_consultar_recibos.png", $this->getDiretorioImagens()) . ' alt="Consultar Recibos" title="Consultar Recibos"/>';
-          $strAcoesProcedimento .= '</a>';
+        $numTabBotao = $objPaginaSEI->getProxTabBarraComandosSuperior();
+        $strAcoesProcedimento .= '<a href="' . $objSessaoSEI->assinarLink('controlador.php?acao=pen_procedimento_estado&acao_origem=procedimento_visualizar&acao_retorno=arvore_visualizar&id_procedimento=' . $dblIdProcedimento . '&arvore=1') . '" tabindex="' . $numTabBotao . '" class="botaoSEI">';
+        $strAcoesProcedimento .= '<img class="infraCorBarraSistema" src=' . ProcessoEletronicoINT::getCaminhoIcone("/pen_consultar_recibos.png", $this->getDiretorioImagens()) . ' alt="Consultar Recibos" title="Consultar Recibos"/>';
+        $strAcoesProcedimento .= '</a>';
       }
     }
 
+    //Apresenta o bot�o de cancelar tr�mite
+    $objAtividadeDTO = $objExpedirProcedimentoRN->verificarProcessoEmExpedicao($objSeiIntegracaoDTO->getIdProcedimento());
+    if ($objAtividadeDTO && $objAtividadeDTO->getNumIdTarefa() == ProcessoEletronicoRN::obterIdTarefaModulo(ProcessoEletronicoRN::$TI_PROCESSO_ELETRONICO_PROCESSO_EXPEDIDO)) {
```

Comment:
Prezado Mauro,
Está faltando fechar este "if", e isso está ocasionando problemas ao subir o ambiente.

Reply:
Prezado, foi corrido o bug gerado durante o merge com master

Reply:
Bom dia Mauro,
Eu fiz o teste ontem e hoje, e vejo que segue o erro. Não consegue rodar a aplicação. O arquivo PENIntegração foi corrigido na tua branch?
Parse error: syntax error, unexpected 'public' (T_PUBLIC) in /opt/sei/web/modulos/pen/PENIntegracao.php on line 172
(mensagem de agora).